common/shared.c

```diff
@@ -1021,6 +1021,9 @@ ssize_t ft_rx(struct fid_ep *ep, size_t size)
 	}
 	/* TODO: verify CQ data, if available */
 
+	/* Ignore the size arg. Otherwise posted receive wouldn't have enough space
+	 * when the benchmark tests increments the send data size. Instead we post
+	 * a size of the entire receive buffer. */
```

Comment:
I need to check, but I remember looking at this a while back, and realizing that the maximum sized recv buffer was posted. The sender can always send a smaller message. @a-ilango can you verify if this is still the case?